packages/plugin/src/theia.d.ts

```diff
@@ -3877,6 +3877,12 @@ declare module '@theia/plugin' {
          */
         iconPath?: string | Uri | { light: string | Uri; dark: string | Uri } | ThemeIcon;
 
+        /**
+         * A human readable string which is rendered less prominent.
+         * When `true`, it is derived from [resourceUri](#TreeItem.resourceUri) and when `falsy`, it is not shown.
+         */
+        description?: string | boolean;
```

Comment:
q: do we need a change log entry for this?

Reply:
I don't think so. It is an addition of an optional property that actually should have been there before.